crates/db_schema/src/impls/tagline.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{
+  newtypes::LocalSiteId,
+  schema::tagline::dsl::*,
+  source::tagline::*,
+  utils::{get_conn, DbPool},
+};
+use diesel::{insert_into, result::Error, ExpressionMethods, QueryDsl};
+use diesel_async::{AsyncPgConnection, RunQueryDsl};
+
+impl Tagline {
+  pub async fn replace(
+    pool: &DbPool,
+    for_local_site_id: LocalSiteId,
+    list_content: Option<Vec<String>>,
```

Comment:
As mentioned above, this should just be Vec<String>

Reply:
If you leave this as option, at least move the if let outside the transaction function. Seems very inefficient to start a new async task etc just to do nothing and return Ok.